### ISSUE TITLE
Render preview for non-runnable assemblies

### ISSUE THREAD

**Issue by ForNeVeR** [ForNeVeR/AvaloniaRider#55]
Currently, when rendering a preview, we rely on `AssemblyName.deps.json` and `AssemblyName.runtimeconfig.json` being available for the assembly containing the XAML file in question. These files aren't available for non-runnable assemblies in .NET Core, so the preview won't work.

For controls in such assemblies, we should provide a runnable project selector: filter only the runnable projects that depend on the current assembly, and preselect one of them by default.

If no such project exists, then we won't be able to show a preview.

**Comment by PieroCastillo** [comment]
About that feature, when can it be available? 'cause in Avalonia VS extension that already exists

**Comment by ForNeVeR** [comment]
@PieroCastillo, it is on my top-priority list. Will be available as soon as possible. I understand the plugin is unusable for most projects without it, yes.

**Comment by wdcossey** [comment]
Already done some work on this @ForNeVeR , I got it working but it was a little ugly at the time.

I am spread pretty thin with work at the moment but an more than happy to upload the code to a forked branch.

**Comment by ForNeVeR** [comment]
Some feedback from users about that feature:
> can you make it use the selected project as the executable (same project it would use for debugging if I press debug button)?
>
> or recommended automatically first time someone loads an avalonia project?

**Comment by ForNeVeR** [comment]
While I haven't finished with the full implementation, yet, there'll be a quick and dirty option for people blocked by the lack of this feature. See #108 for details.
